scripts/s3-deployer/main.go

```diff
@@ -53,12 +53,25 @@ func run() {
 	}
 }
 
+type logger struct{}
+
+func (l *logger) Log(v ...interface{}) {
+	fmt.Printf("AWS Log: %v", v)
+}
+
 func createSession() {
 	// Specify profile to load for the session's config
 	var err error
+	var verboseErr = true
+	var logLevel = aws.LogDebug
+	_ = logLevel
```

Comment:
Do we still need the verbose error reporting?

Reply:
I left it in there so it's easy to toggle if we need it in the future.